frame/session/src/lib.rs

```diff
@@ -122,7 +122,7 @@ use sp_runtime::{
 };
 use sp_staking::SessionIndex;
 use frame_support::{
-	ensure, decl_module, decl_event, decl_storage, decl_error, ConsensusEngineId, Parameter,
+	ensure, ConsensusEngineId, Parameter,
```

Comment:
I think `ensure` can be deleted here? IIRC it's automatically imported from `frame_support::pallet_prelude::*`.

Reply:
Its required here since `ensure` is used outside `mod pallet`, like [here](https://github.com/paritytech/substrate/blob/6bed5b0a872f87334cf5356ee9a0e325f5031d02/frame/session/src/lib.rs#L802)